lib/commands.coffee

```diff
@@ -60,30 +57,46 @@ class Commands
     catch error
       atom.notifications.addError error.message, {detail: error.stack, dismissable: true}
 
-  getMessage: (index) ->
-    messages = @linter.views.messages
-    # Use the dividend independent modulo so that the index stays inside the
-    # array's bounds, even when negative.
-    # That way the index can be ++ an -- without caring about the array bounds.
-    messages[index %% messages.length]
-
   nextError: ->
-    if @index?
-      @index++
-    else
-      @index = 0
-    message = @getMessage(@index)
+    {row, column} = atom.workspace.getActiveTextEditor().getCursorBufferPosition()
```

Comment:
what if `activeTextEditor` is null? It'll throw an `undefined is not a function error`